util.go

```diff
@@ -34,3 +36,19 @@ func getRemoteURLGitLab(org, repo string) (string, error) {
 	}
 	return remoteURL, nil
 }
+
+func getGitHubOrganization(webhookType string, webhookEvent interface{}) (string, error) {
+	switch webhookType {
+	case "pull_request":
+		pr := webhookEvent.(*github.PullRequestEvent)
+		return *pr.Organization.Login, nil
```

Comment:
Doesn't really matter too much, but doesn't the API allow you to use the `pr.GetOrganization.GetLogin()` syntax?

Reply:
Yes, thanks